web/templates/base.html

```diff
@@ -63,7 +63,8 @@
 		<div id="push"></div>
 		</div>
 
-		<footer>
+		{% include 'layout/footer.html' %}
+<!--/.		<footer>
```

Comment:
Why is this commentated out again and not removed? 

Because of the other comments which are nested it is actually messing up the HTML code.